### PR TITLE
Feature/doc pep8

### PR DESCRIPTION
**Pull request by davidkydd**
Apply autoDocString and autoPep8

### CODE REVIEW

File: runner/custom_template_factory.py [hunk 1]
```diff
@@ -289,8 +310,15 @@ def get_scene_file(parameters_file: str) -> str:
 
     return scene_file
 
-def get_dedicated_vm_count(parameters_file: str) -> str:
 
+def get_dedicated_vm_count(parameters_file: str) -> str:
+    """[summary]
+    
+    :param parameters_file: Path of the parameters file
```
Comment (matthew-rollitt) [inline runner/custom_template_factory.py:314]: I like seeing this consistent type of style

File: runner/test_manager.py [hunk 2]
```diff
@@ -252,63 +302,131 @@ def submit_pool(self, batch_service_client: batch.BatchExtensionsClient, templat
                 utils.print_batch_exception(err)
 
     def monitor_pool_and_retry_if_needed(self, batch_service_client: batch.BatchExtensionsClient, image_references: 'List[utils.ImageReference]', test_timeout: datetime, stop_thread, VM_image_URL, VM_OS_type):
+        """
```
Comment (matthew-rollitt) [inline runner/test_manager.py:305]: Thanks for adding these

File: runner/utils.py [hunk 3]
```diff
@@ -521,57 +687,81 @@ def wait_for_job_and_check_result(batch_service_client: batch.BatchExtensionsCli
...
             if not does_task_output_file_exist(batch_service_client, job_id, expected_output):
-                raise ex.JobFailedException(job_id, "Failed to find output {}".format(expected_output))
+                raise ex.JobFailedException(
+                    job_id, "Failed to find output {}".format(expected_output))
         else:
             logger.info("Job [{}] is running".format(job_id))
             time.sleep(5)
 
+
 def timedelta_since(start_time: datetime):
```
Comment (matthew-rollitt) [inline runner/utils.py:719]: doesn't this need a return type like -> datetime.datetime
like has_timedout below

File: runner/custom_template_factory.py [hunk 4]
```diff
@@ -53,14 +56,20 @@ def set_custom_image(in_memory_json_object: str, VM_image_URL: str, VM_image_typ
...
+        del in_memory_json_object["variables"][
+            "osType"]["imageReference"]["sku"]
+        del in_memory_json_object["variables"][
+            "osType"]["imageReference"]["version"]
+        in_memory_json_object["variables"]["osType"][
+            "imageReference"]["virtualMachineImageId"] = VM_image_URL
 
     # TODO For handling windows and centos the nodeAgentSKUId needs to be changed in the json send object
-    # The centos image hasn't been tested so I haven't written logic for handling the VM_image_type
+    # The centos image hasn't been tested so I haven't written logic for
```
Comment (matthew-rollitt) [inline runner/custom_template_factory.py]: this can be removed now

Comment (davidkydd) [reply runner/custom_template_factory.py]: done

File: runner/utils.py (hunk 3 above)
Comment (davidkydd) [reply runner/utils.py:719]: it has one?:
 :return: The timedelta between now and start_time
 :rtype: datetime.timedelta

File: runner/custom_template_factory.py (hunk 1 above)
Comment (davidkydd) [reply runner/custom_template_factory.py:314]: [summary].. oops. fixed

File: runner/test_manager.py (hunk 2 above)
Comment (davidkydd) [reply runner/test_manager.py:305]: autopep8 + autodocstring yay

File: runner/utils.py (hunk 3 above)
Comment (matthew-rollitt) [reply runner/utils.py:719]: method needs a type hint https://docs.python.org/3/library/typing.html

**Pull request merged**